lib/modulesync.rb

```diff
@@ -53,9 +53,9 @@ def self.relative_names(file_list, path)
   def self.managed_modules(config_file, filter, negative_filter)
     managed_modules = Util.parse_config(config_file)
     if managed_modules.empty?
-      $stdout.puts "No modules found in #{config_file}." \
+      warn "No modules found in #{config_file}." \
         ' Check that you specified the right :configs directory and :managed_modules_conf file.'
-      exit
+      exit 1
```

Comment:
what do you think about using a different exit code here? On the long run this might make it easier to automate / parse msync runs? Example from the `puppet agent` man page

```
       •   --detailed-exitcodes: Provide extra information about the run via exit codes; works only if '--test' or '--onetime' is also specified. If enabled, 'puppet agent' uses the following exit codes:

           0: The run succeeded with no changes or failures; the system was already in the desired state.

           1: The run failed, or wasn't attempted due to another run already in progress.

           2: The run succeeded, and some resources were changed.

           4: The run succeeded, and some resources failed.

           6: The run succeeded, and included both changes and failures.
```

Reply:
Interesting idea, ATM my goal was to fail when relevant.
AFAICS, this PR already matches with the `puppet agent` exit codes, as it all the fixed codes are set to `1`  and match with a run failed.